src/js/lib/monster.util.js

```diff
@@ -1171,7 +1171,7 @@ define(function(require) {
 		if (!_.isPlainObject(number)) {
 			throw new TypeError('"number" is not an object');
 		}
-		if (!_.has(number, 'features_available') && !_.has(number, '_read_only.features_available')) {
+		if (!_.has(number, 'phoneNumber') || (!_.has(number, 'features_available') && !_.has(number, '_read_only.features_available'))) {
```

Comment:
Not a fan of checking against `phoneNumber` as that prop is added by the `numbers` common control and is not present on a regular number object.

```suggestion
		if (!_.has(number, 'features_available') && !_.has(number, '_read_only.features_available')) {
```

Objectively, the phone number object should not be muted but you shouldn't care about it a the moment.